### PR TITLE
Add colours to debug-log.

### PR DESCRIPTION
**Pull request by howbazaar**
Show logging in glorious colour.

### CODE REVIEW

File: cmd/juju/commands/debuglog.go [hunk 1]
```diff
@@ -189,22 +189,37 @@ func (c *debugLogCommand) Run(ctx *cmd.Context) (err error) {
...
 		}
-		c.writeLogRecord(ctx.Stdout, msg)
+		c.writeLogRecord(writer, msg)
 	}
 
 	return nil
 }
 
-func (c *debugLogCommand) writeLogRecord(w io.Writer, r api.LogMessage) {
+var SeverityColor = map[string]*ansiterm.Context{
```
Comment (mjs) [inline cmd/juju/commands/debuglog.go]: Is this the same as what's in loggo? Are you duplicating to decouple from the default in loggo?

**Comment by mjs** [comment]
LGTM

File: cmd/juju/commands/debuglog.go (hunk 1 above)
Comment (howbazaar) [inline cmd/juju/commands/debuglog.go]: Due to the api sending strings back for levels.

**Comment by howbazaar** [comment]
$$merge$$

**Pull request merged**